SwiftCompilerSources/Sources/AST/DiagnosticEngine.swift

```diff
@@ -0,0 +1,117 @@
+//===--- DiagnosticEngine.swift -------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+@_exported import ASTBridging
```

Comment:
The API of `DiagnosticEngine` should be a pure swift API. So there should be no need to export bridging stuff

Reply:
Removed `@_exporeted`